c2corg_ui/templates/waypoint/edit.html

```diff
@@ -7,139 +9,774 @@
+                 ng-class="{ 'has-error': editForm.capacity.$touched && editForm.capacity.$invalid, 'has-success': editForm.capacity.$valid && editForm.capacity.$touched }">
+              <label><span translate>capacity</span> *</label>
+              <div class="input-group">
+                <input type="number" name="capacity" ng-model="waypoint.capacity"class="form-control" min="0" />
+                <span class="input-group-addon glyphicon glyphicon-user"></span>
+              </div>
+              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="editForm.capacity.$valid && editForm.capacity.$touched"></span>
+              <span class="glyphicon glyphicon-warning-sign form-control-feedback" ng-show="editForm.capacity.$invalid && editForm.capacity.$touched"></span>
+              <div class="help-block" ng-messages="editForm.capacity.$error">
+                <p ng-message="number"  style="{{ (editForm.capacity.$invalid && editForm.capacity.$touched) ? 'display:inline' : ''}}" translate>This field must be a number.</p>
+              </div>
+            </div>
+
+            ## CAPACITY STAFFED
+            <div class="form-group data half" ng-if="waypoint.waypoint_type == 'gite' || waypoint.waypoint_type == 'camp_site' || waypoint.waypoint_type == 'hut' || waypoint.waypoint_type == 'shelter'"
+                 ng-class="{ 'has-error': editForm.capacity_staffed.$touched && editForm.capacity_staffed.$invalid, 'has-success': editForm.capacity_staffed.$valid && editForm.capacity_staffed.$touched }">
+              <label><span translate>capacity_staffed</span> *</label>
+              <div class="input-group">
+                <input type="number" name="capacity_staffed" ng-model="waypoint.capacity_staffed"class="form-control" min="0" />
+                <span class="input-group-addon glyphicon glyphicon-user"></span>
+              </div>
+              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="editForm.capacity_staffed.$valid && editForm.capacity_staffed.$touched"></span>
+              <span class="glyphicon glyphicon-warning-sign form-control-feedback" ng-show="editForm.capacity_staffed.$invalid && editForm.capacity_staffed.$touched"></span>
+              <div class="help-block" ng-messages="editForm.capacity_staffed.$error">
+                <p ng-message="number"  style="{{ (editForm.capacity_staffed.$invalid && editForm.capacity_staffed.$touched) ? 'display:inline' : ''}}" translate>This field must be a number.</p>
+              </div>
+            </div>
+
+          </div>
+        </section>
+
+        ## MAPS
+        <section class="section maps-info">
+          <h2 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#edit-maps"><span translate>maps_info</span><span class="glyphicon glyphicon-menu-down"></span></h2>
```

Comment:
"Maps info" is not a relevant name for this section. Prefer something like "Geolocation".